web/src/ui/util/useLongTap.test.tsx

```diff
@@ -35,7 +35,7 @@ it("does not trigger for drags", async () => {
   const { getByText } = render(<LongTapTest onLongTap={cb} />);
   const target = getByText("Touch Target");
   await tap(target, { action: "down", pos: { x: 0, y: 0 } });
-  await fireEvent.pointerMove(target, {
+  fireEvent.pointerMove(target, {
```

Comment:
Did this work both with and without and await? What's going on here?

Reply:
Yea, the function doesn't return anything so it was just doing `await undefined` which is a no-op.